packages/datepicker-react/src/dateFunctions.ts

```diff
@@ -1,4 +1,4 @@
-export const dayMonthYearRegex = /^(\d\d)\.(\d\d)\.(\d{4})/;
+export const dayMonthYearRegex = /^(\d\d)[\.-](\d\d)[\.-](\d{4}|\d{2})/;
```

Comment:
Kan du legge til $ tegn på slutten av regex? For eksempel, "12.12.2020abc" blir ikke gyldig.